javagen/src/main/java/com/azure/autorest/mapper/ProxyMethodMapper.java

```diff
@@ -147,7 +147,11 @@ public static ProxyMethodMapper getInstance() {
             for (Parameter parameter : request.getParameters().stream()
                     .filter(p -> p.getProtocol() != null && p.getProtocol().getHttp() != null)
                     .collect(Collectors.toList())) {
-                if (parameter.isRequired() || !settings.isLowLevelClient()) {
+                if (!settings.isLowLevelClient() || parameter.isRequired() &&
+                        parameter.getProtocol().getHttp().getIn() != RequestParameterLocation.Header &&
+                        parameter.getProtocol().getHttp().getIn() != RequestParameterLocation.Query ||
+                        parameter.getLanguage().getJava().getName().equals("accept")) {
+                        // Maybe we should allow constant parameter here, to be confirmed
```

Comment:
Do we also provide proxy method parameter in LLC if the parameter is from client `.getFromClient() == true` (this might help for the `api-version` parameter)?

@jianghaolu

Reply:
Fixed. I missed a pair of `()`.